mix.exs

```diff
@@ -46,7 +46,9 @@ defmodule Boombox.Mixfile do
   defp deps do
     [
       {:membrane_core, "~> 1.1"},
-      {:membrane_webrtc_plugin, "~> 0.23.2"},
+      # {:membrane_webrtc_plugin, "~> 0.23.2"},
+      {:membrane_webrtc_plugin,
+       github: "membraneframework/membrane_webrtc_plugin", branch: "fix-bugs"},
```

Comment:
I have approved this branch, so you can release it and set the dependency to the hex version

Reply:
for now I changed it to github master, I think we should release after the needed fix in elixir_webrtc